test/setup.js

```diff
@@ -8,7 +9,29 @@ import {
     expectKaTeX, expectEquivalent,
 } from "./helpers";
 
-// Serializer support
+// HTML/MathML serializer
+
+// This serializer is based on jest-serializer-html
+// [https://github.com/rayrutjes/jest-serializer-html/blob/master/index.js]
+// but with a hack to prevent strings from getting trimmed, which is important
+// in particular for MathML space tests, but also to ensure we aren't
+// generating extra spaces.
+expect.addSnapshotSerializer({
+    print(val) {
+        const trim = String.prototype.trim;
+        String.prototype.trim = function() { return this; };
+        try {
+            return toDiffableHtml(val);
+        } finally {
+            String.prototype.trim = trim;
+        }
```

Comment:
Is there any other library that can do this?

Reply:
We could search.  But also a new version of this library should now do what we want, thanks to https://github.com/rayrutjes/diffable-html/pull/15